PrReviewCode/Program.cs

```diff
@@ -48,4 +50,6 @@ public static void Main(string[] args)
 
         app.Run();
     }
-}
\ No newline at end of file
+    
+}
```

Comment:
Uncaught exception in /weatherforecast endpoint will crash the endpoint. Ensure this is only present in test scenarios or wrap with try/catch.

 **SUGGESTION:**

```suggestion
 // Add conditional check for test mode or handle via try/catch block
```